apps/api/src/__tests__/mock-services.ts

```
@@ -57,6 +65,11 @@ export class MockAuditTrailService {
     // Mock implementation
     return Promise.resolve()
   }
+
+  static async logSecurityEvent(event: any) {
+    // Mock implementation for test compatibility
+    return Promise.resolve()
+  }
```

Biome (2.1.2)

[error] Duplicate class member name "logSecurityEvent"



(lint/suspicious/noDuplicateClassMembers)